demos/dermatology.html

```diff
@@ -134,13 +134,16 @@
         if (def) {
             def.split("&").forEach(function (param, idx) {
                 var paramParts = param.split("=");
+                if (paramParts[0] === "hpcc_debug") { return; }

```

Comment:
Also it was my understanding that this wasn't really useful in the dermatology, but more for your app?